vt/utils.py

```diff
@@ -0,0 +1,12 @@
+import asyncio
+
+
+def _make_sync(future):
```

Comment:
Since `make_sync` is now used in external modules (client.py and iterator.py), I'd suggest removing its `_` prefix. Citing PEP 8 (https://www.python.org/dev/peps/pep-0008/#descriptive-naming-styles):

> _single_leading_underscore: weak "internal use" indicator. E.g. from M import * does not import objects whose names start with an underscore.

```suggestion
def make_sync(future):
```

Reply:
Updating the PR to reflect this change.